pyface/dock/tests/test_dock_sizer.py

```diff
@@ -0,0 +1,27 @@
+# (C) Copyright 2005-2021 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+
+import unittest
+from unittest.mock import MagicMock
+
+from pyface.dock.dock_sizer import DockControl
+from pyface.toolkit import toolkit
+
+not_wx = toolkit.toolkit != "wx"
+
+class TestDockControl(unittest.TestCase):
+
+    @unittest.skipIf(not_wx, "This test is specific to the wx backend")
```

Comment:
To my surprise, `pyface/dock/dock_sizer.py` is wx specific although it lives in a generic module.https://github.com/enthought/pyface/blob/1a416eeea5146477a52f2dd1dbe6550111c54f2d/pyface/dock/dock_sizer.py#L20